src/app/components/loading-gif/loading-gif.component.scss

```diff
@@ -0,0 +1,6 @@
+.loading {
+    margin: 0 auto;
+    background: url('../../../assets/images/loading-gif.gif') no-repeat center center;
```

Comment:
Any reason we aren't using font awesome for this? That way we aren't bound to the gif-resolution, plus we're alreadty loading font awesome on each page so its also one less request :) see http://fontawesome.io/examples/#animated